<ui version="4.0" >
 <class>CatalogPasswordDialog</class>
 <widget class="QDialog" name="CatalogPasswordDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure passwords</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="passGroupBox" >
     <property name="title" >
      <string>Password protected catalog</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="RPcheckBox" >
            <property name="text" >
             <string>Read password protected</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Re-enter Read Password</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Read Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="rpLineEdit2" />
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="rpLineEdit1" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="WPcheckBox" >
            <property name="text" >
             <string>Write password protected</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Write Passwrord</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Re-enter Write Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="wpLineEdit2" >
              <property name="echoMode" >
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="wpLineEdit1" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CatalogPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CatalogPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
